python/amici/ode_export.py

```diff
@@ -2325,6 +2339,11 @@ def _write_function_file(self, function: str) -> None:
             f'}} // namespace model_{self.model_name}',
         ])
 
+        # check custom functions
+        for fun in CUSTOM_FUNCTIONS:
+            if 'include' in fun and any(fun['c++'] in line for line in lines):
```

Comment:
Right.

Regarding checking for boost, I think this might be somewhat of a pain. For me okay to leave that part, until this really causes problems... A general warning, that model building trouble maybe due to the additional requirement will do the job for me...

Reply:
If that is fine, I would leave out the checking for boost then for the moment. Where should one add the warning that optional dependencies may be the problem (with link to the INSTALL.md)?

Reply:
Something like this? https://github.com/AMICI-dev/AMICI/pull/1222/commits/6ea1364886a868f966fd09eef5d5ada0d83a76fd
Feel free to revert...

Reply:
PS: ideally something like that would be part of the generated model code, so that it is also displayed when building the extension independently of the import, but I guess that would quite some distutils pain again and is probably not used by too many people anyways.